.github/workflows/TEST2.yml

```
@@ -0,0 +1,98 @@
+    strategy:
+      matrix:
+        java-version: [24, 25-ea]  # Test on both JDK 24 and 25 Early Access
+        include:
+          - java-version: 24
+            distribution: 'temurin'
+            experimental: false
+          - java-version: 25-ea
+            distribution: 'oracle'  # Oracle provides EA builds
+            experimental: true
+      fail-fast: false  # Don't cancel other jobs if one fails
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)